### PR TITLE
Collectorset-controller 0.13.0

### PR DESCRIPTION
**Pull request by vkumbhar94**

### CODE REVIEW

File: config-templates/argus/versioned/argus-config-1.0.0.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ etcdDiscoveryToken: ""
 # The image respository of the [Argus](https://hub.docker.com/r/logicmonitor/argus) container.
 imageRepository: logicmonitor/argus
 # The image tag of the [Argus](https://hub.docker.com/r/logicmonitor/argus/tags) container.
-imageTag: v6
+imageTag: v5
```
Comment (kedarjapan) [inline config-templates/argus/versioned/argus-config-1.0.0.yaml:31]: Why is this changed to v5 ? for argus 1.0.0, it should be v6 right ?

Comment (vkumbhar94) [reply config-templates/argus/versioned/argus-config-1.0.0.yaml:31]: Yes it should be v6, correcting this

**Review by kedarjapan — APPROVED** [review]
